apps/mason2/genomic_variants.cpp

```diff
@@ -59,7 +59,7 @@ int StructuralVariantRecord::endPosition() const
             if (size > 0)
                 return pos;
             else
-                return pos + size;
+                return pos + std::abs(size);
```

Comment:
Since we know that size is negative, we could use a cheaper operation. This would also equal what happens in the `nearBreakend` function.
```suggestion
                return pos - size;
```

Reply:
also fine for me